src/scss/Landing.scss

```diff
@@ -129,14 +158,22 @@ $mobile: 500px;
         font-size: 20px;
         font-weight: 300;
         color: #74869b;
+
+        @media screen and (max-width: $mobile) {
+          display: none;
+        }
       }
     }
 
     .tiles {
       display: flex;
       width: 50%;
+      margin-left: -35px;
```

Comment:
I slightly dislike having negative margin adjustments, it just feels "wrong", but I'm not seeing a great alternative right now, and the improvement is there, so.

Reply:
Same and same.